### DISCUSSION TITLE
Speaker Recognition for telephone recordings

### DISCUSSION THREAD

**Discussion by DieguJota** [NVIDIA-NeMo/NeMo#4795] [Q&A]
Hello

I already work with the NeMo transcriber and I have had great results and I want to move on to speaker recognition.
In my last trainings for the Portuguese language, I didn't get good results for the recognition of announcers in telephone calls.

My goal is to find out if a person is really who they are. Just comparing embendding with embendding.

Are there good practices for this training dataSet, targeting this type of work that I intend to do on phone calls?

**Comment by titu1994** [comment]
@nithinraok could you provide some info here

**Comment by nithinraok** [comment]
TitaNet-large model shows good performance on phone and meeting recordings. Can you explain more about what experiments you have done and what cosine similarity scores you were getting? 

if possible please share a couple of audio files for which you would like to compare the embeddings.
